files/splunk_hec_collect_api_events.rb

```diff
@@ -70,61 +53,90 @@ def process_response(body, total, settings, index_file, source_type, splunk_clie
   true
 end
 
-config_file = ENV['CONFIG_FILE'] || "#{confdir}/splunk_hec.yaml"
+STORE_DIR = ENV['STORE_DIR'] || '/etc/puppetlabs/splunk/'
 
-# ensure the config directory is created.
-FileUtils.mkdir_p STORE_DIR
+API_JOBS_STORE = "#{STORE_DIR}/splunk-jobs-store".freeze
+API_ACTIVITY_STORE = "#{STORE_DIR}/splunk-activity-store".freeze
```

Comment:
Just a small nit: can we move the definition of constants like this to the top of the file? When defining constants like this it's customary to do this at the top just after the `requires` statements but before any functions that they depend on are defined.